examples/modal/modal.go

```diff
@@ -149,6 +173,22 @@ func handleModal(w http.ResponseWriter, r *http.Request) {
 		w.WriteHeader(http.StatusUnauthorized)
 		return
 	}
+
+	// update modal sample
+	switch i.Type {
+	//update when interaction type is view_submission
+	case slack.InteractionTypeViewSubmission:
+		api := slack.New("YOUR_TOKEN_HERE")
+		//you can use any modal you want to show to users just like creating modal.
+		updateModal := updateModal()
+		//you can use hash and ViewID from payload and externalID if exists
+		_, err := api.UpdateView(updateModal, "", i.View.Hash, i.View.ID)
+		if err != nil {
+			fmt.Printf("Error updating view: %s", err)
+			w.WriteHeader(http.StatusUnauthorized)
```

Comment:
```suggestion
			w.WriteHeader(http.StatusInternalServerError)
```

Reply:
I see,this should be StatusInternalServerError.